save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/views/organization/OrganizationSettingsMenu.kt

```diff
@@ -141,48 +141,52 @@ private fun organizationSettingsMenu() = FC<OrganizationSettingsMenuProps> { pro
             div {
                 className = ClassName("card card-body mt-0 p-0")
                 if (props.selfRole.isSuperAdmin()) {
-                    div {
-                        className = ClassName("d-sm-flex justify-content-center form-check pl-3 pr-3 pt-3")
+                    props.onCanCreateContestsChange?.let { canCreateContestsChange ->
                         div {
-                            input {
-                                className = ClassName("form-check-input")
-                                type = InputType.checkbox
-                                value = props.organization.canCreateContests.toString()
-                                id = "canCreateContestsCheckbox"
-                                checked = props.organization.canCreateContests
-                                onChange = {
-                                    props.onCanCreateContestsChange(!props.organization.canCreateContests)
+                            className = ClassName("d-sm-flex justify-content-center form-check pl-3 pr-3 pt-3")
+                            div {
+                                input {
+                                    className = ClassName("form-check-input")
+                                    type = InputType.checkbox
+                                    value = props.organization.canCreateContests.toString()
+                                    id = "canCreateContestsCheckbox"
```

Comment:
Move super admin settings from a user settings tab to a separate tab, that will be visible only for super admins